frontend/sentiment-dashboard/src/app/login/login.component.html

```diff
@@ -1,13 +1,11 @@
-
```

Comment:
In the nav component, can we make the "Log Out" link change to "Log In" or "Log Out" based on the current auth state?